src/Rokt-Kit.js

```diff
@@ -0,0 +1,128 @@
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+var roktLauncherScript = 'https://apps.rokt.com/wsdk/integrations/launcher.js';
+
+var name = 'Rokt';
+var moduleId = 181;
+
+var constructor = function () {
+    var self = this;
+
+    self.name = name;
+    self.moduleId = moduleId;
+
+    function initForwarder(settings) {
+        var accountId = settings.accountId;
+        var sandboxMode = settings.sandboxMode === 'True';
+
+        if (!window.Rokt || !(window.Rokt && window.Rokt.currentLauncher)) {
+            var target = document.head || document.body;
+            var script = document.createElement('script');
+            script.type = 'text/javascript';
+            script.src = roktLauncherScript;
+            script.async = true;
+            script.crossOrigin = 'anonymous';
+            script.fetchPriority = 'high';
+            script.id = 'rokt-launcher';
+
+            script.onload = function () {
+                // Once the script loads, ensure the Rokt object is available
+                if (
+                    window.Rokt &&
+                    typeof window.Rokt.createLauncher === 'function' &&
+                    window.Rokt.currentLauncher === undefined
+                ) {
+                    window.Rokt.createLauncher({
+                        accountId: accountId,
+                        sandbox: sandboxMode,
+                    })
+                        .then(function (launcher) {
+                            // Assign the launcher to a global variable for later access
+                            window.Rokt.currentLauncher = launcher;
+                            window.mParticle.Rokt.attachLauncher(launcher);
+                        })
+                        .catch(function (err) {
+                            console.error('Error creating Rokt launcher:', err);
+                        });
+                } else {
+                    console.error(
+                        'Rokt object is not available after script load.'
+                    );
+                }
+            };
+
+            script.onerror = function (error) {
+                console.error('Error loading Rokt launcher script:', error);
+            };
+
+            target.appendChild(script);
+        } else {
+            console.warn('Unable to find Rokt on the page');
+        }
+    }
+
+    this.init = initForwarder;
+};
+
+function getId() {
+    return moduleId;
+}
+
+function register(config) {
+    var forwarderNameWithSuffix = [name, suffix].join('-');
```

Comment:
i'd get rid of any information regarding `suffix` since `suffix` doesn't exist.  braze is in a special category.  you can pull code in this section from anther pre-integration builder that doesn't have suffix, like [amplitude](https://github.com/mparticle-integrations/mparticle-javascript-integration-amplitude/blob/master/src/Amplitude.js#L747-L784)